README.md

```diff
@@ -51,7 +51,7 @@ Ruby on Rails
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://github.com/Alabi12/Rails_BlogApp.git)
+- [Live Demo Link](Not yet)
```

Comment:
- [ ] kindly remove the Live demo section from your README, since you don't yet have a deployed version of your application. You can add it back later on when you do. This will make your README clean and professional